relevanceai/vector_tools/cluster.py

```diff
@@ -683,7 +686,7 @@ def hdbscan_cluster(
                 ".".join([cluster_field, vector_fields[0], alias])
             )
         # load the documents
-        docs = self._get_all_documents(
+        documents = self.get_all_documents(
```

Comment:
It's strange that this changed was reverted.

Reply:
Okay will fix ~